src/backend/utils/adt/agtype_util.c

```diff
@@ -197,20 +197,24 @@ uint32 get_agtype_length(const agtype_container *agtc, int index)
  */
 static int get_type_sort_priority(enum agtype_value_type type)
 {
-    if (type == AGTV_OBJECT)
+    if (type == AGTV_PATH)
```

Comment:
As we are updating a function, please make sure that it [the function] conforms to our standards. In this case, {} for all if/else/for/while/do/etc. So, for all of these ifs and anything else in this function. You don't need to worry about functions that you don't touch.

Reply:
Got it, I will update this right away.

Reply:
Done.